crates/ruff_linter/src/rules/ruff/rules/redundant_bool_literal.rs

```diff
@@ -0,0 +1,129 @@
+use ruff_diagnostics::{Diagnostic, Edit, Fix, FixAvailability, Violation};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::Expr;
+use ruff_python_semantic::analyze::typing::traverse_literal;
+use ruff_text_size::Ranged;
+
+use bitflags::bitflags;
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks for `Literal[True, False]` type annotations.
+///
+/// ## Why is this bad?
+/// The `bool` type has exactly two constant instances: `True` and `False`. Writing
+/// `Literal[True, False]` in the context of type annotations could be replaced by a
+/// bare `bool` annotations. Static type checkers such as [mypy] treat them as
+/// equivalent.
```

Comment:
Start off with a short sentence that clearly states that this is a rule concerned with readability and style rather than correctness:

```suggestion
/// `Literal[True, False]` can be replaced with `bool` in type annotations,
/// which means the same thing but is more concise and more readable. 
///
/// This is because the `bool` type has exactly two constant instances:
/// `True` and `False`. Static type checkers such as [mypy] treat
/// `Literal[True, False]` as equivalent to `bool` in a type annotation.
```